### PR TITLE
usagedocs-readme-update

### PR DESCRIPTION
**Pull request by lxmarinkovic**
Hi @krisnova , I thought these few additions would make sense. Please review and let me know what you think about it :D

### CODE REVIEW

**Comment by quintessence** [comment]
They do! We try to limit the number of rules so they're easy to understand. A few things: 

- We try to limit the number of rules and make them clear and appropriately broad, so that we don't have more rules than a user can reasonably expect to read / follow. (This is an "over time" statement and not directly about adding 1-2 new rules.)
- I am working on a rule explainer to help explain what things we consider to fall under each rule, which should help
- We do have to discuss new rules before pushing them live :)

TLDR - I'm gonna leave your PR open for a bit, but we definitely have seen it and even if we don't incorporate it specifically we will incorporate the intent. Thank you!

**Comment by lxmarinkovic** [comment]
Cool, thanks 😃

**Comment by quintessence** [comment]
PR update (all PRs are getting this update): we're in the process of rolling out our CLA, which we're going to have all contributors sign moving forward. We're (I'm) going to circle back on the PRs once that's complete.

**Comment by quintessence** [comment]
Ok here we go :tada: 

We have finally rolled out our new CLA \o/ The CLA is through the Nivenly Foundation (the non-profit we're spinning up to house Hachyderm and OSS projects). The Docs are being licensed as CC and code (for code work on Hachyderm, Aurae, etc.) is Apache 2.0. (I realize code work doesn't apply here; still need to sign both parts for now :) ). The CLA is signed via a CLA bot that integrates with our GitHub repo: basically all future PRs will validate that you've signed the CLA.

https://cla.nivenly.org/

Relevant note: your signature is private information and will not be shared publicly.

**Comment by arjunrbery** [comment]
@cla-bot check

**Comment by lxmarinkovic** [comment]
Signed CLA 🚀

**Comment by lxmarinkovic** [comment]
> @cla-bot check
